dts/bindings/mtd/nxp,imx-flexspi-mx25um51345g.yaml

```diff
@@ -6,3 +6,14 @@ description: NXP FlexSPI MX25UM51345G
 compatible: "nxp,imx-flexspi-mx25um51345g"
 
 include: ["nxp,imx-flexspi-device.yaml", soc-nv-flash.yaml]
+
+properties:
+  device-base:
+        type: int
+        required: false
+        description: |
+          Base memory mapped address of this flash device. Typically the same
```

Comment:
Do you mind providing an example (not in the PR, but in this thread).  What would the dts look like for this case?

Reply:
I've dropped this property in the latest push, see my comment below